### PR TITLE
Fix 0RTT and 1RTT definitions in PacketType

### LINKED ISSUE #128: Remove implementation detail from PacketType enum
At the moment, the PacketType enum has `onertt = "1RTT"` (and similar for 0RTT) because in most programming languages, enum keys cannot start with numerical characters. This has however led to some confusion in some cases, so it's probably best to leave out this implementation detail in the spec.

See also https://github.com/p-quic/pquic/issues/18#issuecomment-756005163

### PR DESCRIPTION
**Pull request by lnicco**
Fixes #128

### CODE REVIEW

File: draft-ietf-quic-qlog-quic-events.md [hunk 1]
```diff
@@ -1300,8 +1300,8 @@ class IPAddress : string | bytes;
 enum PacketType {
     initial,
     handshake,
-    zerortt = "0RTT",
-    onertt = "1RTT",
+    0RTT,
```
Comment (lnicco) [inline draft-ietf-quic-qlog-quic-events.md:1303]: nit: one could argue that RTT should be lowercase for consistency with the other types.

Comment (rmarx) [reply draft-ietf-quic-qlog-quic-events.md:1303]: Somewhat disagree. RTT is an acronym and the others are not. The RFCs also use the uppercased version of this. Finally, for backwards-compat with existing qlog implementations, 0RTT and 1RTT is easiest.

Comment (lnicco) [reply draft-ietf-quic-qlog-quic-events.md:1303]: @rmarx sounds good. 
Are there any other concerns, or can this be merged?

Comment (rmarx) [reply draft-ietf-quic-qlog-quic-events.md:1303]: This is fine I think. 

For other, larger changes, we'd ideally also add an entry in the changelist on the bottom. But this is such a small thing that it's not needed.

**Pull request merged**